core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -548,6 +548,12 @@ class ColumnFormatRelation(
         throw e
     }
   }
+
+  /** Base table of this relation. */
+  override def baseTable: Option[String] = _origOptions.get("COLOCATE_WITH")
+
+  /** Name of this relation in the catalog. */
+  override def name: String = _table
```

Comment:
The underscore was used to indicate a parameter which is not a field but just passed to constructor. If it is being used in body (and thus will become a member field) then better to remove underscore. Parent class has a public "table" val so better to use that.